packages/ResizeDetector/stories/storyHelpers.js

```diff
@@ -0,0 +1,79 @@
+import React from "react";
+import styled from "styled-components";
+import tokens from "@paprika/tokens";
+import stylers from "@paprika/stylers";
+import Heading from "@paprika/heading";
+import ExternalLink from "@paprika/external-link";
+import { Story, Rule, Tagline } from "storybook/assets/styles/common.styles";
+import { useDimensions, useBreakpoints } from "../src";
+
+export const exampleStoryParameters = {
+  options: {
+    isToolshown: true,
+    showPanel: false,
+  },
+  viewMode: "story",
+};
+
+export const ExampleStory = props => {
+  const { children, fileName, storyName, tagline } = props;
+  return (
+    <Story>
+      <HeaderFlex>
+        <Heading level={1} displayLevel={2} isLight>
+          {storyName}
+        </Heading>
+        {fileName && (
+          <ExternalLink
+            hasNoUnderline
+            href={`https://github.com/acl-services/paprika/blob/master/packages/ResizeDetector/stories/${fileName}`}
+          >
+            Source
+          </ExternalLink>
+        )}
+      </HeaderFlex>
+      {tagline && <Tagline>{tagline}</Tagline>}
+      <Rule />
+      {children}
+    </Story>
+  );
+};
```

Comment:
If everyone thinks this is a good pattern, we could extract it to `/.storybook/components` and use it everywhere.

Reply:
this is good, but I guess we need a page where we can see what component are available for building our stories, right now only a few of us know about this component, and it's easy to forget about them.  (of course for another pr or for future initiatives)

Reply:
There's a section now in the Storybook Navigation for ... `Storybook` components.  Only `<Codeviewer>` and `<ResizeDetector>` there so far, but that's where I think we could show off and document these types of components.